components/lvgl_ili9341/ili9488.c

```diff
@@ -125,13 +126,13 @@ void ili9488_flush(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * col
     }
 #else
     lv_color16_t *buffer_16bit = (lv_color16_t *) color_map;
-    uint8_t mybuf[3 * size * sizeof uint8_t];
+    uint8_t mybuf[3 * size * sizeof(uint8_t)];
```

Comment:
Change to uint8_t* mybuf = (uint8_t*) heap_caps_malloc(3 * size * sizeof(uint8_t), MALLOC_CAP_DMA);

or it will have stack overflow

Reply:
Addressed with ef6be68